src/test/java/com/loadero/models/TestPolling.java

```diff
@@ -165,4 +173,55 @@ public void timeoutZero() {
                 launch.getTestId(), launch.getId(), Duration.ofSeconds(5), Duration.ofSeconds(0));
         });
     }
+
+    @Test
+    @Order(7)
```

Comment:
Unfortunately, yes. Otherwise, Wiremock's stubs get confused which leads to sending wrong requests to the wrong stubs and getting back wrong responses(if getting them at all).